Readme.md

```diff
@@ -1,19 +1,44 @@
+
+This repository represents the official **Catena-X Open Digital Rights Language (ODRL) profile**
+
+The primary aim of the **Catena-X ODRL Profile** is to provide standardized contractual modules such as permissions, prohibitions, obligations, and dispensations in a manner such that they can be checked automatically, in a machine-readable format. In short: our ODRL profile helps us to **automate contractual negotiations** on the Catena-X Data Space as by default, contract negotiations happen electronically via a registered connector. This repository contains the technical definition of the Catena-X ODRL Profile and the legal definitions of the terms used in the Catena-X Data Space during the *Dataspace Protocol Negotiation process* and its resulting agreements.
+
+All standardized contractual modules referenced in the Catena-X ODRL profile have a legal description, a technical term to reference them ("leftOperand") and in some cases a defined list of possible or allowed or standardized values ("rightOperand").
+
+The Catena-X ODRL profile has an initial focus on providing **standardized data usage policies "purposes"** for the Catena-X Use Cases as a mutual foundation that all data space participants can rely on to ensure trust, interoperability, and scalability. These policies represent the Catena-X value proposition of **data sovereignty** in its purest form.
+
+Below we list out an example for such a standardized usage policy based on the official Catena-X ODRL profile (*as of Release 24.05*):
+
+- **Example Catena-X leftOperand** for a standardized Catena-X usage policy: cx-policy:UsagePurpose
+- **Example Catena-X rightOperand** for a standardized Catena-X usage policy: cx.circular.dpp:1
+- **Example legally binding purpose description** of standardized Catena-X RightOperand: "Exchange and use of data according to the applicable public legal regulation directly requiring digital product passports or affecting the contents or handling of digital product passports."
 
 Contributions are made to the ODRL profile itself [profile.ttl](./profile.ttl)
 
 The human readable version is generated from the *.ttl file and stored under [profile.md](./profile.md)
 
+# Context on the Open Digital Rights Language (ODRL)
+
+The Open Digital Rights Language (ODRL) is a policy expression language that provides a flexible and interoperable information model, vocabulary, and encoding mechanisms for representing statements about the usage of content and services. The ODRL Information Model describes the underlying concepts, entities, and relationships that form the foundational basis for the semantics of the ODRL policies.
+
+Policies are used to represent permitted and prohibited actions over a certain asset, as well as the obligations required to be meet by stakeholders. In addition, policies may be limited by constraints (e.g., temporal or spatial constraints) and duties (e.g. payments) may be imposed on permissions.
```

Comment:
I understand that this is coming from ODRL, but should we avoid the "payments" part to not confuse the audience?

Reply:
That I don't understand... where is there a reference to payments here?

Reply:
the text says:
> "limited by constraints (e.g., temporal or spatial constraints) and duties (**e.g. payments**)"
and I just thought, this could confuse our CX audience.

Reply:
Ha, thanks! Fixed!

Reply:
Done